.ci/package.json

```diff
@@ -4,15 +4,18 @@
   "description": "Continuous integration of Zotero translators",
   "main": "index.js",
   "scripts": {
-    "test": "find .. -maxdepth 1 -name '*.js' -print0|xargs -0 -n1 -P8 ./checkTranslator.sh --skip-warn",
-    "start": "find .. -maxdepth 1 -name '*.js' -print0|xargs -0 -n1 -P1 ./checkTranslator.sh"
+    "errors": "find .. -maxdepth 1 -name '*.js' -print0|xargs -0 eslint -c eslintrc --quiet",
+    "test": "find .. -maxdepth 1 -name '*.js' -print0|xargs -0 eslint -c eslintrc",
+    "start": "find .. -maxdepth 1 -name '*.js' -print0|xargs -0 eslint -c eslintrc --max-warnings 0"
```

Comment:
Do we need the `start` line? Seems like `errors` and `test` are sufficient, and it's less clear what `start` should do anyway, so I'd remove this.

Reply:
I don't know -- checkTranslators had several check modes and I didn't know where they were used.

"start" reports warning and errors but will exit with a non-0 exit status if it finds warnings, "test" will report warnings and errors but only exits with non-0 when any errors are found

I added "errors" because there were so many warnings it was hard to see the errors...